src/fenced/fenced/logging.py

```diff
@@ -26,6 +28,19 @@ def handleError(self, record):
             self.sock = None
 
 
+def ensure_logdir_exists():
+    """
+    We need to ensure that the directory in `LOG_FILE` exists
+    so logging works
+    """
+    dirname = os.path.dirname(LOG_FILE)
+
+    if not os.path.exists(dirname):
```

Comment:
this check is probably a no-op, we could just drop it and just try os.makedirs since exist_ok=True anyway? Thoughts?

Reply:
If I don't get `dirname` then `os.makedirs` actually makes a `/root/syslog/fenced.log` directory.

Reply:
No, I mean `if not os.path.exists(dirname):`, we can drop that condition and just do `os.makedirs(dirname, exist_ok=True)`

Reply:
oh derp, will fix.